### ISSUE TITLE
Consider number of browser pods instead of total number of pods - number of Moon pods

### ISSUE THREAD

**Issue by vania-pooh** [aerokube/moon#168]
Google deployer pods should be excluded from the list of moon pods.
```
      'app.kubernetes.io/component': 'deployer.marketplace.cloud.google.com',
```

**Comment by vania-pooh** [comment]
Should use https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/ to fix this.

**Comment by vania-pooh** [comment]
Fixing this for Google Cloud is not needed as we are not providing Moon in marketplace. Pod calculation logic was fixed in Moon 2.0.0.
